doc/development_guide/contributor_guide/tests/install.rst

```diff
@@ -27,6 +27,10 @@ This ensures your testing environment is similar to Pylint's testing environment
 pre-commit hooks which should take care of the autoformatting for you before each
 commit. To enable it, run ``pre-commit install`` in the ``pylint`` root directory.
 
+**Even more optionally**: You can enable slow on push hooks with ``pre-commit install --install-hooks -t pre-push``.
```

Comment:
@DanielNoord here is where you have to opt-in. I think we're good. If a contributor pushes up the changes with py-version, sees it fail again, they'll look at the diff and figure it out.

Reply:
```suggestion
**Even more optionally**: You can enable slow on push hooks with ``pre-commit install --install-hooks -t pre-push``. You will need to use the same python interpreter and enchant dict installed than the CI for it to work.
```

Reply:
Not applying my own suggestion as I'm going to fix it in the script.